src/app_service/service/chat/dto/chat.nim

```diff
@@ -12,6 +12,7 @@ type ChatType* {.pure.}= enum
   Profile = 4,
   Timeline = 5
   CommunityChat = 6
+  Category = 9999
```

Comment:
@richard-ramos the same reason as above, I would remove this and make it as it was and update the rest of the code accordingly to that change.